src/module.c

```diff
@@ -6376,6 +6383,11 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
  *              * VM_BlockClient
  *              * VM_GetCurrentUserName
  *
+ *     * 'X' -- Return exact reply types, including the differences between simple
+ *              string and bulk string, and the RESP 2 difference between nulls
+ *              and null arrays.. This flag is only effective when combined with
```

Comment:
Good description, but there's an extra dot.

```suggestion
 *              and null arrays. This flag is only effective when combined with
```

Why must it be combined with the `E` flag?

Reply:
:man_facepalming:  it doesn't. I was confused for a moment thinking that call reply objects were only created when `E` was set, but `E` is only for error cases.